### DISCUSSION TITLE
Add potion effects

### DISCUSSION THREAD

**Discussion by malekashkar** [HypixelDev/PublicAPI#328] [Ideas]
I think it would be nice to be able to see what potion effects a user has activated and how long it's going to last for. Thank you!

**Comment by dxxxxy** [comment]
They've already said something in the past for player stats such as health and others being updated too frequently and that's why they didn't add them.

**Comment by puppy0cam** [comment]
Potion effects only tick down while the player is online on that profile. So this would indeed be considered as part of the player health being updated too frequently.

**Comment by malekashkar** [comment]
So you guys don't think they're ever going to add them?

I'm guessing the API works as the game sending the data and storing it instead of requesting the data from the game when receiving the call then?

**Comment by puppy0cam** [comment]
The API works by requesting from the database, not the game.
